.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
This should be added in `setup.cfg`, saves the additional file.

Not sure if we want to ignore these two errors globally? What's the reason?

Reply:
I've been using the [black code formatter](https://github.com/ambv/black) on this code.

Black is fully PEP8 compliant, whereas flake8 is not for rules E203 and W503: https://github.com/ambv/black#line-breaks--binary-operators

Reply:
> This should be added in setup.cfg, saves the additional file.

thanks!